http-ui/src/main.rs

```diff
@@ -448,6 +451,15 @@ async fn main() -> anyhow::Result<()> {
                             Setting::NotSet => (),
                         }
 
+                        // We transpose the settings JSON struct into a real setting update.
+                        match settings.sortable_attributes {
+                            Setting::Set(sortable_attributes) => {
+                                builder.set_sortable_fields(sortable_attributes)
+                            }
+                            Setting::Reset => builder.reset_sortable_fields(),
```

Comment:
For the moment you could rebase your branch on https://github.com/meilisearch/milli/pull/330 to use this `reset` method.

Reply:
done